django/contrib/admin/static/admin/css/changelists.css

```diff
@@ -159,7 +159,6 @@
     font-weight: 400;
     padding: 0 15px;
     margin-bottom: 10px;
-    cursor: pointer;
```

Comment:
Does this break the cursor for `#changelist-filter h3`?

I wasn't sure about this yet, hence I hadn't removed it in the other PR.

Reply:
Oh, missed that 🤦🏻

Although, should the `<h3>` have a pointer? I see that there are `<a>` tags within the `<h3>` tags anyway...

Reply:
I don't think so, could you share a bit more what's your thinking?

The H3 under `#changelist-filter` looks like this with this change:
![image](https://github.com/django/django/assets/124304/85249d1f-865a-4d84-b982-65664cef9848)

Reply:
Good point Nick on why those are headers, initially I thought it was so screen readers announce that following the hierarchy:

```
H2 Filters -> H3 Show Counts
          |-> H3 By Author
...
```

But the details/summary in that sidebar are not using H3 :thinking:

Reply:
As far as my testing goes, this does not break the existing `details summary` in the changelist filter, so I'll land and continue the work in PR #17910.